tools/src/main/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommand.java

```diff
@@ -806,12 +806,10 @@ private static void executeMoves(Admin adminClient,
         do {
             Set<TopicPartitionReplica> completed = alterReplicaLogDirs(adminClient, pendingReplicas);
             if (!completed.isEmpty()) {
-                System.out.printf("Successfully started log directory move%s for: %s%n",
-                    completed.size() == 1 ? "" : "s",
-                    completed.stream()
-                        .sorted(ReassignPartitionsCommand::compareTopicPartitionReplicas)
-                        .map(Object::toString)
-                        .collect(Collectors.joining(",")));
+                completed.stream().sorted(ReassignPartitionsCommand::compareTopicPartitionReplicas).forEach(replica -> {
+                    System.out.printf("Successfully started moving log directory to %s for replica %s-%s with broker id: %s %n",
```

Comment:
This seems like a much nicer way to format the output. I suggest that `... with broker %s%n` would be a bit neater.

Reply:
Thank you for the suggestion!  let me revise this